modules/sw360/sw360-workflow/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/processors/SW360Enricher.java

```diff
@@ -58,13 +73,13 @@ public void configure(Map<String, String> configMap) {
     }
 
     SW360MetaDataReceiver createMetaDataReceiver(Map<String, String> configMap) {
-        String sw360ProxyHost = context.getToolConfiguration().getProxyHost();
-        int sw360ProxyPort = context.getToolConfiguration().getProxyPort();
-        SW360ConnectionConfigurationFactory configurationFactory = new SW360ConnectionConfigurationFactory();
         SW360Connection connection =
-                configurationFactory.createConnection(key -> getConfigValue(key, configMap),
-                        key -> getBooleanConfigValue(key, configMap),
-                        sw360ProxyHost, sw360ProxyPort);
+                getConnectionFactory().createConnection(key -> getConfigValue(key, configMap),
+                        context.getHttpClient(), context.getObjectMapper());
         return new SW360MetaDataReceiver(connection);
     }
+
+    SW360ConnectionConfigurationFactory getConnectionFactory() {
```

Comment:
Getter isn't necessary, if this if only used here

Reply:
It is used by tests.